build/dependencies.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AspNetCoreVersion>2.0.0-preview1-*</AspNetCoreVersion>
```

Comment:
weird. This PR is on `dev` branch but the version here is `preview1`...something github is doing?